src/main/java/com/wildfire/main/config/enums/Pronoun.java

```diff
@@ -50,6 +51,24 @@ public String toString() {
 		return subjective + "/" + objective;
 	}
 
+	public static Text local(Pronoun pronoun, Boolean subjective) {
```

Comment:
```suggestion
	public static Text local(Pronoun pronoun, boolean subjective) {
```
unless you're providing this function `null` here you don't need to use the `Boolean` class type over the `boolean` primitive